### ISSUE TITLE
Some utilities don't connect properly when loading from a save

### ISSUE THREAD

**Issue by koosemose** [TeamPorcupine/ProjectPorcupine#1677]
### Expected behavior

When a save is loaded all furniture that is properly connected to a power cable that is connected to a proper power source should have power

### Actual behavior
Some of the furniture doesn't connect properly when loaded from a save.
![image](https://cloud.githubusercontent.com/assets/6334643/21072663/f3c92372-be97-11e6-9ca4-a1921da6f1fc.png)

### Steps to reproduce the behavior

- Step 1: Load a save with furniture connected to power cables

A save that shows this error: https://gist.github.com/koosemose/4e3f15ab18b2c3682729117696d826d3

It may not be all configurations that do this, I am uncertain at this time. I'm also not certain if the issue lies in the saving or loading code, or in utilities or furniture. Any assistance narrowing this down would be appreciated (or a solution if you come across it)

**Comment by koosemose** [comment]
The only bit of information that I've been able to glean so far, is that it doesn't seem to have to do with the order the furnitures or utilities are placed in, I've done swapped both, both independently and together, and it's still always the lower air pump that doesn't have power when loading. 

I've also tried saving with an extra piece of wire both to the left and below the malfunctioning air pump. And loading in and out of dev mode, in case one of those had some kind of effect.
